doc/releases/changelog-dev.md

```diff
@@ -153,6 +153,10 @@
 
 <h3>Improvements</h3>
 
+* `qml.matrix` now can also compute the matrix of tapes/QNodes that contain multiple
+  broadcasted operations, as they are present in decompositions of broadcasted operations.
```

Comment:
```suggestion
  broadcasted operations and/or the decompositions of broadcasted operations.
```

Reply:
I had already rewritten it a bit, to be more explicit in what I meant. Does this work now?